JS/app.js

```diff
@@ -35,89 +55,125 @@ Store.prototype.randomCookieValue = function () {
   var tTotalEl=document.createElement('td');
   tTotalEl.textContent = this.totalForAllHours;
   tRowEl.appendChild(tTotalEl);
-  //  
   tableEl.appendChild(tRowEl);
+};
+//All stores call
+var renderAllCookieRow = function() {
+  for(var i; i<5; i++) {
+    allStores[i].renderCookieRow();
+  }
+};
 
-  tRowEl = document.createElement('tr');
-  //sets blank cell
-  var tHoursRow = document.createElement('td');
-  tHoursRow.textContent = '';
-  tRowEl.appendChild(tHoursRow);
-  // fills in hours
-  for(i=0; i<hours.length; i++){
-    tHoursRow = document.createElement('td');
-    tHoursRow.textContent = allStores[i] ;
-    tRowEl.appendChild(tHoursRow);
+var renderCookieFooter= function() {
+  // creates first block in table(Hard coded)
+  tRowEl_2 = document.createElement('tr');
+  var tStoreNameEl = document.createElement('tfoot');
+  tStoreNameEl.textContent = "total for hour";
+  tRowEl.appendChild(tStoreNameEl);
+  // for generates cookie amounts over all hours
+  for(var i = 0; i < hours.length; i++) {
+    var tDataEl = document.createElement('tfoot');
+    tDataEl.textContent = totalArrPerHour [i];
+    tRowEl.appendChild(tDataEl_2);
   }
-  tHoursRow = document.createElement('td');
-  tHoursRow.textContent = allStores;
-  tRowEl.appendChild(tHoursRow);
-  //attach to table
+  // finishes with last column which is sum of hours
+  var tTotalEl=document.createElement('td');
+  tTotalEl.textContent = this.totalForAllHours;
+  tRowEl.appendChild(tTotalEl);
   tableEl.appendChild(tRowEl);
+  
+Store.addNewStore= function(event) {
+//prevent refreshing of page
+  event.preventDefault();
+  var newLocation = event.target.storeLocation.value;
+  var newMinSales = event.target.minimumSales.value;
+  var newMaxSales = event.target.maximumSales.value;
+  var newAvgSales = event.target.avgSales.value;
+
+  new Store(newLocation, newMinSales, newMaxSales, newAvgSales);
 };
-// create new object for constructor object
+// tRowEl = document.createElement('tr');
+
+// var tHoursRow = document.createElement('td');
+// tHoursRow.textContent = '';
+// tRowEl.appendChild(tHoursRow);
+// // fills in hours
+// for(i=0; i<hours.length; i++){
+//   tHoursRow = document.createElement('td');
+//   // tHoursRow.textContent = allStores[i] ;
+//   tRowEl.appendChild(tHoursRow);
+// }
+// tRowEl.appendChild(tHoursRow);
+// //attach to table
+// tableEl.appendChild(tRowEl);
+
+
 
 var firstAndPike = new Store('First and Pike',23,65,6.3);
 var seaTacAirport = new Store('SeaTac Airport',3,24,1.2);
 var seaCenter = new Store('Seattle Center',11,38,2.3);
 var capHill= new Store(' Capitol Hill',20,38,2.3);
-var alki= new Store('Alki',2,16,4.6);
\ No newline at end of file
+var alki= new Store('Alki',2,16,4.6);
+
+renderHeaderRow();
+//  --- need to get this working 
+// renderAllCookieRow();
+firstAndPike.renderCookieRow();
+seaTacAirport.renderCookieRow();
+seaCenter.renderCookieRow();
+capHill.renderCookieRow();
+alki.renderCookieRow();
+new Store.renderCookieRow();
```

Comment:
This might be a problem spot. It looks like you are trying to create a new Store object instance & render it's data to the DOM. Alternatively, you could grab the new instance from your stores array once it is created from your input form.